doc/varnish-configuration.rst

```diff
@@ -174,6 +205,17 @@ Tagging
 
 Feature: :ref:`cache tagging <tags>`
 
+For this feature you'll need to choose between either xkey, or BAN based tags invalidation.
```

Comment:
we should repeat the core difference here and link to the section above that introduces xkey.